### ISSUE TITLE
Pickers: Custom document picker focusZone keyboarding issues

### ISSUE THREAD

**Issue by kristoferbrown** [OfficeDev/office-ui-fabric-react#1695]
### Bug Report

- __Package version(s)__: current
- __Browser and OS versions__: Windows Chrome and Edge, likely all browsers

#### Describe the issue:

The document list doesn't support proper keyboarding. 

#### Actual behavior:

You can only focus the first document and the inner buttons are inaccessible.

#### Expected behavior:

All documents in the list and all inner control buttons should be accessible via keyboard.

### If applicable, please provide a codepen repro:

Repros on current website.

**Comment by kristoferbrown** [comment]
This has been fixed by recent picker and focus zone updates.
